test/reconstruction/gapfill_minimum_reactions.jl

```diff
@@ -2,15 +2,9 @@
     #=
     Implement the small model that should be gapfilled.
     =#
-    model = StandardModel("partialmodel")
-    m1 = Metabolite("m1")
-    m2 = Metabolite("m2")
-    m3 = Metabolite("m3")
-    m4 = Metabolite("m4")
-    m5 = Metabolite("m5")
-    m6 = Metabolite("m6")
-    m7 = Metabolite("m7")
-    m8 = Metabolite("m8")
+    model = StandardModel("partial model")
+
+    (m1, m2, m3, m4, m5, m6, m7, m8) = Metabolite.("m$i" for i = 1:8)
```

Comment:
wow, didn't know this is possible

Reply:
I actually wanted to just use the array later in reactions but apparently `m[1] → m[2]` syntax doesn't work there.

Reply:
that would be nice actually... later we need to rethink how the user manually enters reactions and metabolites, to make it better